test/lib/wovnrb_test.rb

```diff
@@ -247,6 +248,72 @@ def test_call_with_path_ignored_should_not_change_environment
     assert_call_affects_env(settings, env, mock_api: false, affected: false)
   end
 
+  def test_call__with_use_cookie_lang_true__cookie_lang_is_target_lang__should_redirect
```

Comment:
need a few more test cases. What we want to cover is:
- Request is target lang (cookie lang is another target lang), does not redirect
- Request is default lang
    - no cookie defined, no redirect
    - cookie defined but no feature, no redirect
    - cookie also set to default lang, no redirect
    - cookie set to target lang, redirect

Reply:
These should now be covered in the latest changes.